deploy/helm/sumologic/values.yaml

```diff
@@ -89,6 +89,10 @@ sumologic:
     # Source category for the Events source. Default: "{clusterName}/events"
     sourceCategory: ""
 
+  # Traces configuration
```

Comment:
Can we make sure we add a comment indicating this is an alpha/closed beta feature and not intended for use by everyone? Probably work with Pawel on the verbage.

Reply:
+1